Makefile

```diff
@@ -72,3 +72,16 @@ logsf:
 dbclient:
 	@echo "Opening DB client"
 	docker-compose run php drupal database:client
+
+behat:
+	@echo "Running behat tests"
+	docker-compose run php vendor/bin/behat
+
+phpcs:
+	@echo "Running coding standards on custom code"
+	docker-compose run php vendor/bin/phpcs --standard=vendor/drupal/coder/coder_sniffer/Drupal web/modules/custom --ignore=*.min.js --ignore=*.min.css
```

Comment:
I tried to run this command, and it error'ed me with this message:
zsh: no matches found: --ignore=*.min.js

Is the command correct? Or maybe I'm doing something wrong. I tried it on the move.mil project

Reply:
I can't reproduce, in theory this should work with move.mil, but I hadn't tested it lets debug when you have a minute